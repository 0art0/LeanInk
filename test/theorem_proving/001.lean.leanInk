[{"contents": "theorem test (p q : Prop) (hp : p) (hq : q) : p ∧ q ∧ p :=\n  ",
  "_type": "text"},
 {"messages":
  [{"contents":
    "test/theorem_proving/001.lean:2:2: warning: declaration uses 'sorry'\n",
    "_type": "message"}],
  "goals": [],
  "contents": "sorry",
  "_type": "sentence"},
 {"contents": "", "_type": "text"}]