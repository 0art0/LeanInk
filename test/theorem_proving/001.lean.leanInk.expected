[{"contents": "theorem test (p q : Prop) (hp : p) (hq : q) : p ∧ q ∧ p :=\n  ",
  "_type": "text"},
 {"messages":
  [{"contents": "Warning: declaration uses 'sorry'", "_type": "message"}],
  "goals": [],
  "contents": "sorry",
  "_type": "sentence"},
 {"contents": "", "_type": "text"}]